1_fetch/src/fetch_wqp_data.R

```diff
@@ -54,7 +54,9 @@ fetch_wqp_data <- function(site_counts_grouped, characteristics, wqp_args = NULL
                                    characteristicName = c(characteristics)))
   
   # Pull data
-  wqp_data <- dataRetrieval::readWQPdata(wqp_args_all)
+  wqp_data <- retry::retry(dataRetrieval::readWQPdata(wqp_args_all),
+                           when = "Error:", 
+                           max_tries = 3)
```

Comment:
Should we consider making `max_tries` a parameter that the user can adjust?

Reply:
Good idea, I've added `max_tries` as an argument in `fetch_wqp_data()` (with a default of `max_tries = 3`).